### ISSUE TITLE
Python code completion is not working

### ISSUE THREAD

**Issue by jealcalat** [quarto-dev/quarto-vscode#57]
Code completion is not working on Python chunks.

This is the expected behavior 

![image](https://user-images.githubusercontent.com/35947922/184272664-0836a168-fd8e-4bc8-b2fc-0b176c809d56.png)

But I got this

![image](https://user-images.githubusercontent.com/35947922/184272829-fcbde4ef-b865-4eda-830e-f9867eebd2cf.png)

Completion works just fine in `.py` scripts

![image](https://user-images.githubusercontent.com/35947922/184272848-b1e38efb-05f0-4968-a92d-ffba55c677f9.png)

My settings for Quarto in settings.json are

```
  "[quarto]": {
    "editor.suggest.showSnippets": true,
    "editor.wordWrap": "on",
    "editor.quickSuggestions": {
      "comments": "on",
      "strings": "on",
      "other": "on"
    },
    "editor.quickSuggestionsDelay": 250,
    "editor.snippetSuggestions": "top",
    "editor.wordBasedSuggestions": true,
    "editor.suggestOnTriggerCharacters": true,
    "editor.unicodeHighlight.ambiguousCharacters": false,
    "editor.unicodeHighlight.invisibleCharacters": false
  }
```

**Comment by eitsupi** [comment]
duplicate of #10

**Comment by jjallaire** [comment]
This should be working now (fix here: https://github.com/quarto-dev/quarto-vscode/commit/1755d70ca197bdd68413be9b6a81c8133bbd21b3). This is available now in the v1.56.0 release.
